docs/src/dev/runtime_config.md

```diff
@@ -10,6 +10,19 @@ Variables prefixed with `YK_` allow the user to control aspects of yk's executio
 
 The following environment variables are available:
 
+* `YK_LOG=[<path>:]<level>` specifies where, and how much, general information
+  yk will log during execution.
+
+  If `<path>:` (i.e. a path followed by ":") is specified then output is sent
+  to that path. The special value `-` (i.e. a single dash) can be used for
+  `<path>` to indicate stderr. If not specified, logs to stderr.
```

Comment:
One for another day: maybe we should make paths in other env vars (e.g. YKD_LOG_IR) optional too for consistency?

Reply:
Wouldn't hurt but agreed: another PR.